mailbox/store/src/test/java/org/apache/james/mailbox/store/PreDeletionHooksTest.java

```diff
@@ -153,4 +169,14 @@ void runHooksShouldExecuteHooksSequentially() {
             .describedAs("RunHook does not throw if hooks are executed in a sequential manner")
             .doesNotThrowAnyException();
     }
+
+    @Test
+    void runHooksShouldPublishTimerMetrics() {
```

Comment:
could we check that we actually publish a time metric that makes sense (having a sleep of 1 second and check that value)?